### ISSUE TITLE
Add AMP Support for IDT2 Includes

### ISSUE THREAD

**Issue by FK78** [bbc/simorgh#6563]
**Is your feature request related to a problem? Please describe.**
Currently we render IDT2 content on Canonical pages only, we'd like to add support for rendering these Includes on AMP pages as well.

In order to render IDT2 content on AMP pages, we need to output a responsive flat image of the content with an alt attribute. Metadata for IDT2 images is available on the Smallprox /amp endpoint. Data for 3 image sizes is always provided for any IDT2 Include: small, medium and large. These translate to the following screen sizes (as defined by VJ during the development of Smallprox):
- Small = mobile
- Medium = tablet
- Large = desktop

_Note that we should take notice of 'small', 'medium' or 'large' rather than the specific width and height pixel dimensions, as these may change._

This information (required to create this image for AMP) is now included in the AReS response, based on data retrieved from the Smallprox /amp 'mutator' (following work completed in ARES-1007).

**Describe the solution you'd like**
Our solution should be using data from the ARES payload, we should render images based on that data and depending on the breakpoint information/screen size.

_Note: This should only be done for the AMP implementation, we should not be replacing the Canonical implementation._

If there is no image available in the ARES data for any reason, then we will not render anything for the IDT2 include on AMP.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

- There is an a11y requirement to ensure the alt text provided with the IDT2 image metadata is announced on screen readers, so this will need an a11y swarm when complete to check this.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
There is no fallback/error message for IDT2 Includes, if it does not render or no image is available in the ARES response for any reason, then we do not show anything.

Example metadata
![screenshot-1](https://user-images.githubusercontent.com/34196381/82319290-ce2c4600-99c0-11ea-85ee-18beb7e49096.png)
